okhttp/src/main/java/com/squareup/okhttp/MediaType.java

```diff
@@ -31,6 +31,64 @@
   private static final Pattern PARAMETER = Pattern.compile(
       ";\\s*(?:" + TOKEN + "=(?:" + TOKEN + "|" + QUOTED + "))?");
 
+  /**
+   * A {@link MediaType} constant representing the wildcard media type.
+   */
+  public static final MediaType WILDCARD = new MediaType("*/*", "*", "*", null);
```

Comment:
Are you anticipating using `MediaType` in an `Accept` header? At the moment we don't implement media range matching.

https://github.com/square/okhttp/issues/782

I'd prefer to omit this until we support that.